runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorRequest.java

```diff
@@ -0,0 +1,14 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import org.corfudb.format.Types.WorkflowType;
+import org.corfudb.protocols.wireprotocol.ICorfuPayload;
+
+/**
+ * A message container that all orchestrator requests should extend
+ *
+ * Created by Maithem on 10/25/17.
+ */
+
+public abstract class OrchestratorRequest {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "OrchestratorRequest" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)